CAN-D/Src/gps.c

```diff
@@ -70,31 +71,42 @@ void APP_GPS_BufferGPSString(char* dataString, size_t dataLength)
   */
 void APP_GPS_MonitorTask(void const* argument)
 {
-    uint8_t usbTxCnt = 0;
+    GPSData gpsRx = { 0 };
+    volatile uint8_t usbTxCnt = 0;
     osEvent event;
-    GPSData* data;
+    GPSData* gpsRxMsg = &gpsRx;
+    size_t usbMaxMsgLen = GPS_USB_DATA_SZ_BYTES + 10; // Max length of the serialized data
```

Comment:
Maybe we could have a `PROTO_BUFFER_OVERHEAD` defined in the proto handler header